#! /bin/sh /usr/share/dpatch/dpatch-run
## 12_desktop.dpatch by Wolfram Quester <wolfi@sigxcpu.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove FilePattern form inkscape.desktop to get rid of the lintian warning
## DP: desktop-entry-contains-deprecated-key usr/share/applications/inkscape.desktop:91 FilePattern 

@DPATCH@

--- inkscape-0.46.orig/inkscape.desktop.in	2009-04-20 21:44:20.000000000 +0200
+++ inkscape-0.46/inkscape.desktop.in	2009-04-20 21:44:46.000000000 +0200
@@ -10,7 +10,6 @@
 
 MimeType=image/svg+xml
 
-FilePattern=inkscape
 Exec=inkscape %F
 TryExec=inkscape
 Terminal=false
